src/components/CreateCoursePage/index.jsx

```diff
@@ -17,14 +17,26 @@ class CreateCoursePage extends React.Component {
   }
 
   handleCourseCreate(options) {
+    let startDate = '';
+    let endDate = '';
+    if (Date.parse(options.start)) {
```

Comment:
Is this necessary? If the key exists on our backend I'd assume it's a valid date. Not against it just curious about the extra defensive programming here

Reply:
I made this field required again so it should work now without this code.  This was added while testing some error states and not including dates.